notary/src/origo.rs

```diff
@@ -101,11 +124,10 @@ pub async fn sign(
 
   // Log a verifier session for public inputs
   debug!("inserting with session_id={:?}", query.session_id);
-  state
-    .verifier_sessions
-    .lock()
-    .unwrap()
-    .insert(query.session_id.to_string(), VerifierInputs { request_messages, response_messages });
+  state.verifier_sessions.lock().unwrap().insert(query.session_id.to_string(), VerifierInputs {
+    request_messages:  request_messages.clone(),
```

Comment:
why needing a clone here?

Reply:
Good catch, it's not needed. Probably an artifact of some refactor I missed.